ElectronClient/app/gui/note-viewer/preload.js

```diff
@@ -5,8 +5,8 @@
 
 const ipcRenderer = require('electron').ipcRenderer;
 
-ipcRenderer.on('setHtml', (event, html) => {
-	window.postMessage({ target: 'webview', name: 'setHtml', data: { html: html } }, '*');
+ipcRenderer.on('setHtml', (event, html, theme) => {
```

Comment:
Again please make the third parameter an "options" object with a "theme" property. Also it seems this theme only applies to code blocks? In that case, please call it "codeTheme" from the start and everywhere it's used.